src/modules/sale/api.ts

```diff
@@ -46,9 +49,16 @@ const extendedApi = api.injectEndpoints({
             return;
           }
 
+          // TODO: Modify redirect and logic below once "sale has ended" page is created
+          // redirect to sold out page if the sale has ended
+          if (isSaleEnded) {
+            if (!isSoldoutCurrentPage) window.location.pathname = "sold-out";
+            return;
+          }
+
           // sale is sold out, redirect to sold out page
-          if (isSoldout && !isSoldoutCurrentPage) {
-            window.location.pathname = "sold-out";
+          if (isSoldout) {
```

Comment:
`isSaleEnded` and `isSoldout` could be combined but i decided not to since `isSaleEnded` will change in the short future and it will be its own conditional